vm/rust/src/lib.rs

```diff
@@ -124,6 +132,7 @@ pub extern "C" fn cairoVMCall(
 #[derive(Deserialize)]
 pub struct TxnAndQueryBit {
     pub txn: StarknetApiTransaction,
+    pub txn_hash: TransactionHash,
```

Comment:
we need to add this to the json we pass from go side, right?

Reply:
Nice catch, thanks.